goodtablesio/app.py

```diff
@@ -43,6 +44,8 @@
 app.register_blueprint(github)
 app.register_blueprint(s3)
 
+# Set exception logging
+Sentry(app, dsn=settings.SENTRY_DSN)
```

Comment:
Can you put it under `# Register Flask integrations` and use the `sentry.init_app()` form for consistency?